### DISCUSSION TITLE
Update DXCC-Data fails

### DISCUSSION THREAD

**Discussion by VolkerMue** [magicbug/Cloudlog#797] [General]
Hello Community!

Thanks first of all for the great opportunity of your own logbook.
But now I have the first problems with it. 

For days I'm looking for a possible error on my part. I do not get the DXCC updates made. 
Already tried several times for hours, but no DXCC updates are loaded.

The file permissions for FTP folders are set, partly to test with "all 777". As soon as I press the button for DXCC update, the status.html is created in the folder "updates". But from here on nothing happens....no incoming files, etc..

Maybe someone has a solution or suggestions for me.

Greetings and 73
Volker

**Comment by magicbug** [comment]
Could be a couple of things I'd first suggest checking Apache / PHP logs to see if there's anything obvious. The country file is massive in its contents so it can be a bit of a headache depending on server setup.

**Reply by VolkerMue** [reply]
After press "Update DXCC Data" the status.html will be created with 644 permission. 
But the error-log says, that status.html does not exist.

`AH00128: File does not exist: /usr/www/users/vmdchi/cloudlog/updates/status.html, referer: https://dg1vm.de/index.php/update`

**Comment by OM-Bort** [comment]
Since I have updated to the latest version of Cloudlog, I can no longer "Update DXCC Data" either.
After several attempts, I have completely reinstalled Cloudlog. Unfortunately, it still doesn't work and gets stuck at "Dxcc prefixes".
Are there any new findings as to why this could be? There were never any problems with the old version.

**Comment by magicbug** [comment]
I'd suggest enabling logging /application/config/config.php and setting it to the highest level then trying again to see why its failing

You'll find the log files in /application/logs

**Comment by OM-Bort** [comment]
No errors, it simply freezes at the following display:

Updating...
Dxcc Entities: 402
Dxcc Exceptions: 26220
Dxcc Prefixes: 0

Here is the log:
INFO - 2024-01-06 12:04:05 --> Database Driver Class Initialized
INFO - 2024-01-06 12:04:05 --> Session: Class initialized using 'files' driver.
INFO - 2024-01-06 12:04:05 --> Migrations Class Initialized
INFO - 2024-01-06 12:04:05 --> Language file loaded: language/german/migration_lang.php
INFO - 2024-01-06 12:04:05 --> Database Forge Class Initialized
INFO - 2024-01-06 12:04:05 --> Model "Options_model" initialized
INFO - 2024-01-06 12:04:05 --> Model "User_options_model" initialized
INFO - 2024-01-06 12:04:05 --> Controller Class Initialized
DEBUG - 2024-01-06 12:04:05 --> Migration class already loaded. Second attempt ignored.
INFO - 2024-01-06 12:04:12 --> Model "DXCC_Entities" initialized
INFO - 2024-01-06 12:04:13 --> Model "DXCC_Exceptions" initialized

**Reply by magicbug** [reply]
Ah PHP is timing out its quite a big file to process, are you using shared hosting?

**Reply by OM-Bort** [reply]
Yes, it is a shared host.
As mentioned above, there were no problems with the previous version, the DXCC - update was completed within seconds.

I have set the timeout to 999 seconds.

**Reply by OM-Bort** [reply]
Are there no new findings?
Am I the only one with this problem?

**Reply by MikleBas** [reply]
I just checked at home - there are no problems (although due to sanctions we sometimes experience miracles with network connections..)
Try updating files via CRON - Help has how to set this up, if of course this is what you asked about.
73!

**Reply by sforBSD** [reply]
I have the same problem.
No information in logfiles.

status.html created with 644 perms

Last 15 minutes (from click update button) in access.log this appears:

every ~1.5 minutes:
GET status.html 200

and every 2 seconds
GET [...]/index.php/radio/status/

Platform: HP Thin client/FreeBSD/lighttpd/FastCGI/PHP8/Cloudlog 2.6.3

**Reply by sforBSD** [reply]
INFO - 2024-02-13 10:09:12 --> Database Driver Class Initialized
INFO - 2024-02-13 10:09:12 --> Session: Class initialized using 'files' driver.
INFO - 2024-02-13 10:09:12 --> Migrations Class Initialized
INFO - 2024-02-13 10:09:12 --> Language file loaded: language/english/migration_lang.php
INFO - 2024-02-13 10:09:12 --> Database Forge Class Initialized
INFO - 2024-02-13 10:09:12 --> Model "Options_model" initialized
INFO - 2024-02-13 10:09:12 --> Model "User_options_model" initialized
INFO - 2024-02-13 10:09:12 --> Controller Class Initialized
INFO - 2024-02-13 10:09:12 --> Model "User_Model" initialized
INFO - 2024-02-13 10:09:12 --> Model "Cat" initialized
INFO - 2024-02-13 10:09:12 --> Final output sent to browser
INFO - 2024-02-13 10:09:12 --> Total execution time: 0.2014

and repeat the same block every few seconds

**Reply by sforBSD** [reply]
PROBLEM SOLVED.
The problem was caused by missing php lib: simplexml.

**Reply by OM-Bort** [reply]
Unfortunately, I am not an IT expert. Could you please explain where you inserted or added this lib?
I have cPanel here and simplexml is switched on in the PHP settings.

**Reply by K9bdh** [reply]
was there any fix for this.    am  attempting to install wave log and I get same error. it says simplexml is up to date

Brad

**Reply by magicbug** [reply]
Being as your using wavelog which isn't cloudlog I'd suggest seeking help from them
